zcash_client_backend/src/data_api.rs

```diff
@@ -235,11 +243,17 @@ pub trait SaplingInputSource {
 
 /// A trait representing the capability to query a data store for unspent transparent UTXOs
 /// belonging to a wallet.
-#[cfg(feature = "transparent-inputs")]
```

Comment:
It hasn't moved; it has to be removed for the deserialization to be able to deserialize the transparent bits of a proposal.

It can be implemented as a no-op, but the interface has to be public.